telosnetwork/app-telos-native
# Feature/governance documentation

# Addresses  #297 

## Description
Adds governance documentation page

## Test scenarios

1. visit app.telos.native/governance 
2. see IPFS documentation rendered
3. clicking 'amended by' username should redirect to profile
4. clicking section hash link should redirect directly to ipfs page displaying raw data

## Checklist:
